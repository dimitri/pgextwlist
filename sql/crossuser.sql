CREATE ROLE evil_user;
ALTER DATABASE contrib_regression OWNER TO mere_mortal;

SET ROLE mere_mortal;
CREATE TABLE mere_table (t citext);

SET ROLE evil_user;
DROP EXTENSION citext;
DROP EXTENSION citext CASCADE;
