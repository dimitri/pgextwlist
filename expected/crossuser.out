CREATE ROLE evil_user;
ALTER DATABASE contrib_regression OWNER TO mere_mortal;
SET ROLE mere_mortal;
CREATE TABLE mere_table (t citext);
SET ROLE evil_user;
DROP EXTENSION citext;
ERROR:  Permission denied to drop extension, role evil_user is not authorized on this database
DROP EXTENSION citext CASCADE;
ERROR:  Permission denied to drop extension, role evil_user is not authorized on this database
